packages/shared/src/request/post.ts

```diff
@@ -0,0 +1,16 @@
+import request from 'graphql-request';
+import { EmptyResponse } from '../graphql/emptyResponse';
+import { BAN_POST_MUTATION, DELETE_POST_MUTATION } from '../graphql/posts';
+import { apiUrl } from '../lib/config';
+
+export const banPost = (id: string): Promise<EmptyResponse> => {
```

Comment:
I didnt want to make the main components bloated, so I created this folder/files to keep the request helpers

Reply:
Technically also could sit in the graphQL layer right?

Reply:
Yeap. I am fine with both. Either here or in graphql. Just dont want to bloat the component

Reply:
Quite liked the fact graphQL had all the functions related to it.
So maybe we should just standardize it 🙏

Reply:
Makes sense, done 👍